rs/rosetta-api/tests/system_tests/common/utils.rs

```diff
@@ -54,3 +54,20 @@ pub async fn wait_for_rosetta_block(
     }
     panic!("Failed to sync with the ledger");
 }
+
+pub fn bytebuf_to_u64(bytebuf: &[u8]) -> Option<u64> {
```

Comment:
Since this implementation has some strong assumptions (little-endian bytes, only using the first 8), and the function is (currently) only used in one place, would it make sense to name it something like `memo_bytebuf_to_u64`?